### PR TITLE
Add option to unload individual entities

### PR DESCRIPTION
**Pull request by brg468**
Fix for https://github.com/JoshuaMulliken/ha-wyzeapi/issues/332

Tested and no longer returns the error mentioned above.

### CODE REVIEW

File: src/wyzeapy/services/camera_service.py [hunk 1]
```diff
@@ -110,7 +110,3 @@ async def turn_off_notifications(self, camera: Camera):
         ]
 
         await self._set_property_list(camera, plist)
-
-    def register_updater(self, camera: Camera, interval):
-        dev_updater = DeviceUpdater(self, camera, interval)
-        BaseService._update_manager.add_updater(dev_updater)
```
Comment (brg468) [inline src/wyzeapy/services/camera_service.py:116]: I don't think this is being used anymore. The camera and switch platforms forward this to the method in `base_service`

File: src/wyzeapy/services/update_manager.py [hunk 2]
```diff
@@ -67,9 +67,10 @@ async def update_next(self):
             # First we get the next updater off the queue
             updater = heappop(self.updaters)
             # if the updater has been removed, pop the next and clear it from the removed updaters
-            while self.check_if_removed(updater):
-                updater = heappop(self.updaters)
-                self.removed_updaters.remove(updater)
+            if self.removed_updaters:
+                while self.check_if_removed(updater):
+                    self.removed_updaters.remove(updater)
+                    updater = heappop(self.updaters)
```
Comment (brg468) [inline src/wyzeapy/services/update_manager.py:73]: Not necessary as it was working before but this stops calling the `check_if_removed `function on every update cycle if there's nothing to check.

Comment (JoshuaMulliken) [reply src/wyzeapy/services/update_manager.py:73]: Good addition 👍

File: src/wyzeapy/services/camera_service.py (hunk 1 above)
Comment (JoshuaMulliken) [reply src/wyzeapy/services/camera_service.py:116]: Have you tested this against switches and cameras?

Comment (brg468) [reply src/wyzeapy/services/camera_service.py:116]: Against cameras yes, I don't have any switches

Comment (JoshuaMulliken) [reply src/wyzeapy/services/camera_service.py:116]: Ok cool so I will publish as a prerelease so we can get some testing before stable

**Pull request merged**